GraphWebApi/Controllers/KnownIssuesController.cs

```diff
@@ -42,7 +43,8 @@ public async Task<IActionResult> GetKnownIssues()
                                             _knownIssuesTraceProperties);
 
             List<KnownIssue> result = await _knownIssuesService.QueryBugsAsync();
-            return result == null ? NotFound() : Ok(result);
+
+            return result == null ? NotFound() : Ok(JsonConvert.SerializeObject(result));
```

Comment:
Because of this line in the project config
https://github.com/microsoftgraph/microsoft-graph-devx-api/blob/ff1f170bd821fddf4f6800f537e01a8a9ec6c677/GraphWebApi/Startup.cs#L85

I suspect this file can remain unchanged after adding the property attributes and there is no need to call `JsonConvert.SerializeObject`. 
```suggestion
            return result == null ? NotFound() : Ok(result);
```